src/components/Prayer.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+
+const Prayer = () => {
+  const [city, cityState] = useState("gaza strip");
```

Comment:
It will be more declarative if use `setCity`
```suggestion
  const [city, setCity] = useState("gaza strip");
```

Reply:
we put Gaza strip as an initial value